src/reanimated2/commonTypes.ts

```diff
@@ -94,30 +94,93 @@ export type MapperRegistry = {
   stop: (mapperID: number) => void;
 };
 
-export type Context = Record<string, unknown>;
+type WorkletClosure = Record<string, unknown>;
 
-export interface WorkletFunction {
-  __closure?: Context;
+interface WorkletInitDataRelease {
+  code: string;
+}
+
+interface WorkletInitDataDev {
+  code: string;
+  location: string;
+  sourceMap: string;
+  version: string;
+}
```

Comment:
Since this is the addition of only one prop I'd rather have them separately without extending to simplify the dependency chain. If at some point in time worklets would get more properties then we can go into extending I think.